aphrodite/modeling/layers/sampler.py

```diff
@@ -318,12 +318,18 @@ def _apply_temperatures(
     normalized_entropies = dynatemp_entropies.div_(dynatemp_max_entropies)
     dyn_temp = (dynatemp_mins + (dynatemp_maxs - dynatemp_mins) *
                 normalized_entropies.pow_(dynatemp_exps))
-
     temperatures[dynatemp_mask] = dyn_temp
-    temperatures[temperatures <= 0.0] = 1.0
-    # Use float32 to apply temp.
-    # Use in-place division to avoid creating a new tensor.
-    logits = logits.to(torch.float)
+  
+    # There isn't a "safe" temperature range for fp16 logits.
+    # This value was chosen because 1/2e-5 is just under the 65k fp16 max,
+    # meaning that this temperature well-uses the fp16 space after the offset.
+    temperatures[temperatures <= 2e-5] = 2e-5
```

Comment:
Instead of hardcoding it here, can we document it as a constant (e.g. `_MIN_TEMPERATURE = 2e-5`) at the top of the module?

Reply:
That makes sense.